code/game/gamemodes/nuclear/nuclear_challenge.dm

```diff
@@ -42,8 +42,8 @@
 	var/war_declaration = "[user.real_name] has declared his intent to utterly destroy [station_name()] with a nuclear device, and dares the crew to try and stop them."
 	var/custom_threat = alert(user, "Do you want to customize your declaration?", "Customize?", "Yes", "No")
 	if(custom_threat == "Yes" && (world.time < CHALLENGE_TIME_LIMIT-600))
-		war_declaration = sanitize(input(user, "Insert your custom declaration", "Declaration")as text | null)
-		if(war_declaration == null)
+		war_declaration = copytext(sanitize(input(user, "Insert your custom declaration", "Declaration")as text | null), 1, 500)
+		if(war_declaration == null || war_declaration == "")
 			return
```

Comment:
Won't this leave all sorts of variables set as if they declared war, even though we're returning?